docs/SOURCES.md

```diff
@@ -1,255 +1,1188 @@
+| Precipitation Duration    | ❌         | Moon Phase        | ❌         |
+| Wind                      | ✅         | Cloud Cover       | ❌         |
+| Humidity                  | ✅         | Visibility        | ✅         |
+| Dew Point                 | ✅         | Ceiling           | ❌         |
+</details>
+
+## Secondary weather sources
+
+### Atmo AURA
+**[Atmo Auvergne-Rhône-Alpes](https://www.atmo-auvergnerhonealpes.fr/)** provides air quality information for the French region of Auvergne-Rhône-Alpes. This source can be added as a secondary **Air Quality** source.
+
+### ClimWeb
+**[ClimWeb](https://github.com/wmo-raf/climweb)** is an open source content management system developed by WMO Africa for 17 of its member states:
+
+| Country/Territory                 | Agency                                     |
+|-----------------------------------|--------------------------------------------|
+| 🇧🇯 Benin                        | [Météo Benin](http://www.meteobenin.bj/)   |
+| 🇧🇫 Burkina Faso                 | [ANAM-BF](https://www.meteoburkina.bf/)    |
+| 🇧🇮 Burundi                      | [IGEBU](https://www.igebu.bi/)             |
+| 🇹🇩 Chad                         | [Météo Tchad](https://www.meteotchad.org/) |
+| 🇨🇩 Democratic Republic of Congo | [Mettelsat](https://www.meteordcongo.cd/)  |
+| 🇪🇹 Ethiopia                     | [EMI](https://www.ethiomet.gov.et/)        |
+| 🇬🇲 Gambia                       | [DWR](https://meteogambia.org/)            |
+| 🇬🇭 Ghana                        | [GMet](https://www.meteo.gov.gh/)          |
+| 🇬🇼 Guinea-Bissau                | [INM](https://www.meteoguinebissau.org/)   |
+| 🇲🇼 Malawi                       | [DCCMS](https://www.metmalawi.gov.mw/)     |
+| 🇲🇱 Mali                         | [Mali-Météo](https://malimeteo.ml/)        |
+| 🇳🇪 Niger                        | [DMN](https://www.niger-meteo.ne/)         |
+| 🇸🇨 Seychelles                   | [SMA](https://www.meteo.gov.sc/)           |
+| 🇸🇸 South Sudan                  | [SSMS](https://meteosouthsudan.com.ss/)    |
+| 🇸🇩 Sudan                        | [SMA](https://meteosudan.sd/)              |
+| 🇹🇬 Togo                         | [Météo Togo](https://www.anamet-togo.com/) |
+| 🇿🇼 Zimbabwe                     | [MSD](https://www.weatherzw.org.zw/)       |
+
+These sources can be added as a secondary **Alert** and **Temperature normals** source for their respective countries.
+
+### GeoNames
+> 🔐 **This source requires an API key.** [Register here](https://www.geonames.org/login)
+
+**[GeoNames](https://www.geonames.org/)** provides multilingual search for place names of more than 11 million locations worldwide. This source can be enabled as a **Search** source after adding your API key.
+
+### Recosanté
+**[Réseau National de Surveillance Aérobiologique](https://www.pollens.fr/)** (Recosanté) provides pollen information for France. This source can be added as a secondary **Pollen** source.
```

Comment:
I'm not sure it is very clear, but Recosanté and RNSA are two different entities.

RNSA provides detailed pollen information.
This information is then processed by Recosanté to generate an index (so we lose the pollen concentration in the process). Recosanté is a more general health recommendations app.

We use Recosanté and not RNSA directly because RNSA doesn't provide their data for free (only to municipalities).

Reply:
Oh I see. In that case I will just simplify it to something like:

Recosanté can be added as a secondary Pollen source for France.

Feel free to suggest different wordings too.